src/game.js

```diff
@@ -0,0 +1,55 @@
+import { renderRaceResult, renderWinner } from './render.js';
+
+function generateRandomNumber(min, max) {
+  return Math.floor(Math.random() * max) + min;
+}
+
+function playDice() {
+  const randonNumber = generateRandomNumber(1, 9);
+
+  if (randonNumber >= 4) {
+    return true;
+  }
+  return false;
```

Comment:
```suggestion
  return generateRandomNumber(1, 9) >= 4
```
한 줄로 줄일수있는 bool 반환 함수인것 같습니다!

Reply:
넘 좋네요. 굿굿~!!!